src/style.css

```diff
@@ -0,0 +1,133 @@
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  line-height: 18px;
+}
+
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+ 
+.card__image {
+  margin: 32px auto 40px;
+  width: 160px;
+  height: 134px;
+}
+
+h1 {
+  margin: 0;
+}
+
+.card__title {
+  margin: 0 16px;
+  color: #060b35;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+}
+
+.card__product {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+  margin: 4px 16px 16px;
+}
+
+.card__raiting {
+  display: flex;
+  justify-content: space-between;
+  margin: 0 16px 24px;
+}
+
+p {
+  margin: 0;
+}
+
+.star {
+  display: flex;
+  align-items: flex-end;
+  justify-content: space-between;
+  margin: 0 16px 24px;
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+
+}
+
+.stars__star {
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat ;
+  width: 16px;
+  height: 16px;
+  margin: 0 2px;
+  background-position: center;
+}
+
+.stars--4 :nth-child(-n+4) {
+  background-image: url(./images/star-active.svg);
+}
+
+.reviews {
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+}
+
+.card__price {
+  margin: 0 16px 16px;
+  display: flex;
+  align-items: center;
+  /* flex-direction: row; */
+  justify-content: space-between;
+  font-weight: 400;
+}
+
+.card__value {
+  font-weight: 400;
```

Comment:
```suggestion
```
redundant it is default value

Reply:
I took all the values font-weight of 400;

I wanted to ask, is the smallest font always considered the main one?